compose/install/linux.md

```diff
@@ -0,0 +1,141 @@
+---
+description: How to install Docker Compose on Linux
+keywords: compose, orchestration, install, installation, docker, documentation
+toc_max: 3
+
+title: Install on Linux
+redirect_from:
+- /compose/compose-plugin/
+- /compose/compose-linux/
+
+---
+
+On this page you can find instructions on how to install the Compose on Linux from the command line.
+
+## Install Compose
+
+To install Compose:
+* Option 1: [Set up Docker's repository on your Linux system](#install-using-the-repository).
+* Option 2: [Install Compose manually](#install-the-plugin-manually).
+
+### Install using the repository
+
+> **Note**
+>
+> These instructions assume you already have Docker Engine and Docker CLI installed and now want to install the Compose plugin.  
+For Compose standalone, see [Install Compose Standalone](./other.md#on-linux).
+
+If you have already set up the Docker repository, jump to step 2.
+
+1. Set up the repository. Find distro-specific instructions in:
+[Ubuntu](../../engine/install/ubuntu.md/#set-up-the-repository) |
+[CentOS](../../engine/install/centos.md/#set-up-the-repository) |
+[Debian](../../engine/install/debian.md/#set-up-the-repository) |
+[RHEL](../../engine/install/fedora.md/#set-up-the-repository) |
+[SLES](../../engine/install/sles.md/#set-up-the-repository).
+To upgrade, do:
+
+* Ubuntu, Debian:
+
+    ```console
+
+    $ sudo apt-get update
+    $ sudo apt-get install docker-compose-plugin
+
+    ```
+RPM-based distros:
+
+    ```console
+
+    $ sudo yum update
+    $ sudo yum install docker-compose-plugin
+
+    ```
+
+2. Update the `apt` package index, and install the _latest version_ of Docker Compose:
+
+    Ubuntu, Dei:
+
+    ```console
+    $ sudo apt-get update
+    $ sudo apt-get install docker-compose-plugin
+    ```
+
+3.  Verify that Docker Compose is installed correctly by checking the version.
+
+    ```console
+    $ docker compose version
+    Docker Compose version vN.N.N
+    ```
+Where `vN.N.N` is a placeholder for the latest version.
+
+> **Note**
+>
+> To run Compose as a non-root user, see [Manage Docker as a non-root user](../../engine/install/linux-postinstall.md){:target="_blank" rel="noopener" class="_"}.
+
+#### Update Compose
+
+To update, do:
```

Comment:
Are we missing something here? Should this be, "To update Compose, run the following command" or something on those lines?